src/main/java/frc/robot/commands/CmdArmSetExtensionPosition.java

```diff
@@ -0,0 +1,24 @@
+package frc.robot.commands;
+
+import java.util.function.DoubleSupplier;
+
+import edu.wpi.first.wpilibj2.command.InstantCommand;
+import frc.robot.subsystems.Arm;
+
+public class CmdArmSetExtensionPosition extends InstantCommand 
+{
+    private final Arm            _armSubsystem;
+    private final Double _extensionPosition;
```

Comment:
Fix the whitespace on line 10, and use `double` instead of `Double` for line 11 (and also for the parameter in the constructor.

Reply:
This still needs to be fixed.